ui/app/AppLayouts/Chat/stores/UsersStore.qml

```diff
@@ -3,9 +3,21 @@ import QtQuick 2.15
 QtObject {
     id: root
 
+    property var chatCommunitySectionModule
+    property var chatDetails
     property var usersModule
 
-    readonly property var usersModel: usersModule ? usersModule.model : null
+    readonly property var usersModel: {
```

Comment:
The `usersModule.model` provides all contacts and the `chatCommunitySectionModule.membersModel` provides a list of community members?

Reply:
Pretty much. This is a change I did lately where I put a User's module in the section module itself, to hold all the members of the community.

The reason for that being all the public channels of a community contain **all** the members of a community and so we were duplicating the same member model in all of them, and they were big.

So when a community channel is public (no token permissions), we instead use the section's member model. It saves memory and also improves performance when switching from one public channel to another, because the model doesn't change.